lib/tokenize.cpp

```diff
@@ -10968,8 +10968,16 @@ void Tokenizer::simplifyAttribute()
             Token *functok = nullptr;
             if (Token::Match(after, "%name%|*")) {
                 Token *ftok = after;
-                while (Token::Match(ftok, "%name%|::|<|>|* !!("))
+                while (Token::Match(ftok, "%name%|::|<|* !!(")) {
+                    if (ftok->str() == "<") {
+                        Token *tmp = ftok->link();
+                        if (tmp == nullptr)
```

Comment:
Thanks! If the `Token::link` does not work yet.. I think it's better you always use just `ftok->findClosingBracket()`.

    if (ftok->str() == "<") {
         ftok = ftok->findClosingBracket();
         if (!ftok)
              break;
    }

Reply:
Sure, works for me.  Fixed in dbcee87.